rollup.config.js

```diff
@@ -6,7 +6,7 @@ import dts from 'rollup-plugin-dts';
 // We use esbuild for building the actual code.
 
 export default {
-  input: 'out/.dts/mod.d.ts',
+  input: 'out/.dts/src/mod.d.ts',
```

Comment:
When I added files in `perf/` to the `tsconfig.json` include, this intermediate output file moved.

Reply:
OK.